packages/manager/apps/key-management-service/src/pages/dashboard/generalInformations/GeneralInformations.tsx

```diff
@@ -26,7 +26,7 @@ function GeneralInformationsTab() {
   }
 
   return (
-    <div className="grid xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-6">
+    <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 md:gap-6">
```

Comment:
what about mobile view ?

Reply:
classes without prefix are applied on mobile, so it's a `grid-cols-1` setting.